ail_vru_uc1_scenario/cdasim_config/stop_simulation

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Script to stop simulation.
+# Pulls all Kafka message data
+# Stops and removes all docker containers
+# Prunes all Kafka related volumes
+if ! [ -f ./collect_kafka_logs.py ] ; then
+  echo "Installing collect_kafka_logs.py ...."
+  wget -q https://raw.githubusercontent.com/usdot-fhwa-stol/carma-streets/release/lavida/collect_kafka_logs.py
+fi
+filename=$(date +%F_%H-%M-%S)
+python3 collect_kafka_logs.py --zip True "${filename}"
```

Comment:
This could easily lead to `collect_kafka_logs.py` scripts littering a user's environment. I think it would be better to expect this script in `/opt/carma/analytics` (or a similar path) and check for it before invoking the script. We then instruct users to install the script if they want to gather Kafka logs.

Reply:
Hmm, to install this script do I just need to add it to the path. I am ok with making this change as long as it does not significantly expand the scope of the story. Will try something.

Reply:
For this script, I think it would be sufficient to hardcode the path:

```shell
python3 /opt/carma/analytics/collect_kafka_logs.py --zip True "${filename}"
```

Reply:
I added it to the path just for this script. Should not impact the anything else. Does that work?

Reply:
Did that